docs/reference/configuration.md

```diff
@@ -318,9 +318,11 @@ area: {
 
 **Type**: `number` (for `decodeSingle` only)
 
-**Default**: Original image dimensions (when omitted)
+**Default**: `800` (for `decodeSingle`); original image dimensions when using `init()`
 
-**Description**: When using `decodeSingle` with file paths, scales the input image so that the longest side (width or height) equals this value, maintaining aspect ratio. When omitted, the original image dimensions are used without any scaling.
+**Description**: When using `decodeSingle` with file paths, scales the input image so that the longest side (width or height) equals this value, maintaining aspect ratio.
+
+**Important**: `decodeSingle` has a built-in default of `size: 800`. This means images are **automatically scaled to 800px** on their longest side (both larger and smaller images are scaled to match this value) unless you explicitly specify a different value. The `box`, `boxes`, and `line` coordinates in the result are returned in the **scaled coordinate space**, not the original image dimensions. To use the original image size without any scaling, set `inputStream.size` to `0`.
```

Comment:
There should be a link provided here to the documentation section on how to handle scaling box, boxes, and line.